Client/Client.csproj

```diff
@@ -13,6 +13,13 @@
     <NoWarn>1701;1702;NU1701</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(SelfContainedRelease)' == 'true'">
```

Comment:
If we are running `dotnet publish -p:SelfContainedRelease=true`, then this will build a compressed, self-contained single file (native libraries and support files still copied separately), with ready-to-run optimizations (native code generated for some methods to avoid initial JIT overhead).